scripts/analytics.js

```diff
@@ -1,9 +1,9 @@
 // This dotenv import is required for the `.env` file to be read
 require('dotenv').config();
 
-const flexIntegrationSdk = require('sharetribe-flex-integration-sdk');
+const sharetribeIntegrationSdk = require('sharetribe-flex-integration-sdk');
 
-const integrationSdk = flexIntegrationSdk.createInstance({
+const integrationSdk = sharetribeIntegrationSdk.createInstance({
```

Comment:
I think it would be nice to have the same rate limiter SDK initialization code in all example scripts, so that no matter which one people take as starting point, they'll have it initialized correctly.

Reply:
Makes sense, fixed now!